site/source/docs/getting_started/FAQ.rst

```diff
@@ -17,7 +17,7 @@ Why do I get multiple errors building basic code and the tests?
 
 All the tests in the :ref:`Emscripten test suite <emscripten-test-suite>` are known to build and pass on our test infrastructure, so if you see failures locally it is likely that there is some problem with your environment. (Rarely, there may be temporary breakage, but never on a tagged release version.)
 
-First call ``emcc -v``, which runs basic sanity checks and prints out useful environment information. If that doesn't help, follow the instructions in :ref:`verifying-the-emscripten-environment`.
+First call ``emcc -v``, prints out useful environment information. If that doesn't help, follow the instructions in :ref:`verifying-the-emscripten-environment`.
```

Comment:
Another key doc I think we should update is the one linked to here, which goes to a section titled "Sanity tests", in `site/source/docs/building_from_source/verify_emscripten_environment.rst`. That should not mention `-v` any more, and please add a mention of `EMCC_DEBUG=1` in the env.

Reply:
I was thinking of just leaving that sections as `emcc -v` .. since that will actually run the sanity checks the first time around.  

I guess maybe I will add a new option because its kind of annoying to ask folks to set the environment variable here.